core/arch/arm/plat-sprd/console.c

```diff
@@ -43,9 +44,3 @@ void console_putc(int ch)
 	cons->ops->putc(cons, ch & 0xff);
 }
```

Comment:
Could you add a comment that this platform specific function differs from the generic one as `\n` ar not pre-fixed with a `\r`? Maybe the platform maintainer will want to resync.